api/pkg/ivs/stream.go

```diff
@@ -0,0 +1,22 @@
+package ivs
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/ivs"
+)
+
+type GetStreamParams struct {
+	ChannelArn string
+}
+
+func (c *client) GetStream(ctx context.Context, params *GetStreamParams) (*ivs.GetStreamOutput, error) {
+	in := &ivs.GetStreamInput{
+		ChannelArn: &params.ChannelArn,
```

Comment:
AWSパッケージ側で変換用のメソッド用意してくれてるので、こっち使う方が後のライブラリ仕様変更に対応しやすそう

```suggestion
		ChannelArn: aws.String(params.ChannelArn),
```